Makefile

```diff
@@ -7,6 +7,7 @@
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
 GV="network:v1alpha1 servicemesh:v1alpha2 tenant:v1alpha1 tenant:v1alpha2 devops:v1alpha1 iam:v1alpha2 devops:v1alpha3 cluster:v1alpha1 storage:v1alpha1 auditing:v1alpha1 types:v1beta1 quota:v1alpha2 application:v1alpha1 notification:v2beta1"
+MANIFESTS="application/* cluster/* devops/* iam/* network/v1alpha1 quota/* storage/* tenant/*"
```

Comment:
@kubesphere/sig-devops We don't need to generate DevOps's manifest here, am I right?

Reply:
Yes. The DevOps part already moved into https://github.com/kubesphere/ks-devops/

Reply:
@anhoder Thanks for your contribution. This PR looks good to me. A small change is needed since the devops api is no longer used here.

Reply:
alright